### PR TITLE
[Inception V1]change validation to center crop

### PR DESCRIPTION
**Pull request by qiuxin2012**
Current code's Top1 result is 67.204%, while bigdl can reach 68.x %.
Change validation's RandomCrop to CenterCrop can fix this.
The top1 result is below:
```
2019-02-18 14:17:58 INFO  DistriOptimizer$:152 - [Epoch 1 871552/1281167][Iteration 619][Wall Clock 1762.839761811s] Top1Accuracy is Accuracy(correct: 352, count: 50000, accuracy: 0.00704)
2019-02-18 14:47:34 INFO  DistriOptimizer$:152 - [Epoch 2 463232/1281167][Iteration 1239][Wall Clock 3538.710446394s] Top1Accuracy is Accuracy(correct: 1373, count: 50000, accuracy: 0.02746)
2019-02-18 15:17:19 INFO  DistriOptimizer$:152 - [Epoch 3 54912/1281167][Iteration 1859][Wall Clock 5323.887021443s] Top1Accuracy is Accuracy(correct: 3216, count: 50000, accuracy: 0.06432)
2019-02-18 15:47:00 INFO  DistriOptimizer$:152 - [Epoch 3 927872/1281167][Iteration 2479][Wall Clock 7064.733559964s] Top1Accuracy is Accuracy(correct: 6177, count: 50000, accuracy: 0.12354)
2019-02-18 16:16:51 INFO  DistriOptimizer$:152 - [Epoch 4 519552/1281167][Iteration 3099][Wall Clock 8894.933378643s] Top1Accuracy is Accuracy(correct: 8970, count: 50000, accuracy: 0.1794)
2019-02-18 16:46:45 INFO  DistriOptimizer$:152 - [Epoch 5 111232/1281167][Iteration 3719][Wall Clock 10688.954393377s] Top1Accuracy is Accuracy(correct: 11744, count: 50000, accuracy: 0.23488)
2019-02-18 17:16:44 INFO  DistriOptimizer$:152 - [Epoch 5 984192/1281167][Iteration 4339][Wall Clock 12448.424435036s] Top1Accuracy is Accuracy(correct: 13428, count: 50000, accuracy: 0.26856)
2019-02-18 17:46:40 INFO  DistriOptimizer$:152 - [Epoch 6 575872/1281167][Iteration 4959][Wall Clock 14283.433804989s] Top1Accuracy is Accuracy(correct: 15126, count: 50000, accuracy: 0.30252)
2019-02-18 18:16:53 INFO  DistriOptimizer$:152 - [Epoch 7 167552/1281167][Iteration 5579][Wall Clock 16096.245657345s] Top1Accuracy is Accuracy(correct: 17493, count: 50000, accuracy: 0.34986)
2019-02-18 18:47:23 INFO  DistriOptimizer$:152 - [Epoch 7 1040512/1281167][Iteration 6199][Wall Clock 17886.192948501s] Top1Accuracy is Accuracy(correct: 18644, count: 50000, accuracy: 0.37288)
2019-02-18 19:17:50 INFO  DistriOptimizer$:152 - [Epoch 8 632192/1281167][Iteration 6819][Wall Clock 19754.348412693s] Top1Accuracy is Accuracy(correct: 20016, count: 50000, accuracy: 0.40032)
2019-02-18 19:48:29 INFO  DistriOptimizer$:152 - [Epoch 9 223872/1281167][Iteration 7439][Wall Clock 21591.17984033s] Top1Accuracy is Accuracy(correct: 20756, count: 50000, accuracy: 0.41512)
2019-02-18 20:19:12 INFO  DistriOptimizer$:152 - [Epoch 9 1096832/1281167][Iteration 8059][Wall Clock 23393.489167397s] Top1Accuracy is Accuracy(correct: 21211, count: 50000, accuracy: 0.42422)
2019-02-18 20:49:22 INFO  DistriOptimizer$:152 - [Epoch 10 688512/1281167][Iteration 8679][Wall Clock 25244.744184072s] Top1Accuracy is Accuracy(correct: 21990, count: 50000, accuracy: 0.4398)
2019-02-18 21:19:32 INFO  DistriOptimizer$:152 - [Epoch 11 280192/1281167][Iteration 9299][Wall Clock 27054.426171093s] Top1Accuracy is Accuracy(correct: 22744, count: 50000, accuracy: 0.45488)
2019-02-18 21:50:43 INFO  DistriOptimizer$:152 - [Epoch 11 1153152/1281167][Iteration 9919][Wall Clock 28884.51425888s] Top1Accuracy is Accuracy(correct: 22929, count: 50000, accuracy: 0.45858)
2019-02-18 22:21:05 INFO  DistriOptimizer$:152 - [Epoch 12 744832/1281167][Iteration 10539][Wall Clock 30747.809793965s] Top1Accuracy is Accuracy(correct: 23907, count: 50000, accuracy: 0.47814)
2019-02-18 22:51:29 INFO  DistriOptimizer$:152 - [Epoch 13 336512/1281167][Iteration 11159][Wall Clock 32573.238614206s] Top1Accuracy is Accuracy(correct: 24230, count: 50000, accuracy: 0.4846)
2019-02-18 23:21:24 INFO  DistriOptimizer$:152 - [Epoch 13 1209472/1281167][Iteration 11779][Wall Clock 34328.289804483s] Top1Accuracy is Accuracy(correct: 24641, count: 50000, accuracy: 0.49282)
2019-02-18 23:51:13 INFO  DistriOptimizer$:152 - [Epoch 14 801152/1281167][Iteration 12399][Wall Clock 36157.34612081s] Top1Accuracy is Accuracy(correct: 25111, count: 50000, accuracy: 0.50222)
2019-02-19 00:20:54 INFO  DistriOptimizer$:152 - [Epoch 15 392832/1281167][Iteration 13019][Wall Clock 37938.183512043s] Top1Accuracy is Accuracy(correct: 25468, count: 50000, accuracy: 0.50936)
2019-02-19 00:50:23 INFO  DistriOptimizer$:152 - [Epoch 15 1265792/1281167][Iteration 13639][Wall Clock 39668.323456622s] Top1Accuracy is Accuracy(correct: 25956, count: 50000, accuracy: 0.51912)
2019-02-19 01:20:03 INFO  DistriOptimizer$:152 - [Epoch 16 857472/1281167][Iteration 14259][Wall Clock 41487.496946805s] Top1Accuracy is Accuracy(correct: 25943, count: 50000, accuracy: 0.51886)
2019-02-19 01:49:42 INFO  DistriOptimizer$:152 - [Epoch 17 449152/1281167][Iteration 14879][Wall Clock 43266.468745402s] Top1Accuracy is Accuracy(correct: 26244, count: 50000, accuracy: 0.52488)
2019-02-19 02:19:36 INFO  DistriOptimizer$:152 - [Epoch 18 40832/1281167][Iteration 15499][Wall Clock 45056.507440351s] Top1Accuracy is Accuracy(correct: 26419, count: 50000, accuracy: 0.52838)
2019-02-19 02:50:14 INFO  DistriOptimizer$:152 - [Epoch 18 913792/1281167][Iteration 16119][Wall Clock 46851.884415103s] Top1Accuracy is Accuracy(correct: 26748, count: 50000, accuracy: 0.53496)
2019-02-19 03:20:33 INFO  DistriOptimizer$:152 - [Epoch 19 505472/1281167][Iteration 16739][Wall Clock 48716.625140673s] Top1Accuracy is Accuracy(correct: 27152, count: 50000, accuracy: 0.54304)
2019-02-19 03:50:50 INFO  DistriOptimizer$:152 - [Epoch 20 97152/1281167][Iteration 17359][Wall Clock 50533.706301455s] Top1Accuracy is Accuracy(correct: 27030, count: 50000, accuracy: 0.5406)
2019-02-19 04:20:52 INFO  DistriOptimizer$:152 - [Epoch 20 970112/1281167][Iteration 17979][Wall Clock 52296.788305289s] Top1Accuracy is Accuracy(correct: 27459, count: 50000, accuracy: 0.54918)
2019-02-19 04:50:50 INFO  DistriOptimizer$:152 - [Epoch 21 561792/1281167][Iteration 18599][Wall Clock 54133.544249084s] Top1Accuracy is Accuracy(correct: 27675, count: 50000, accuracy: 0.5535)
2019-02-19 05:20:50 INFO  DistriOptimizer$:152 - [Epoch 22 153472/1281167][Iteration 19219][Wall Clock 55933.856234462s] Top1Accuracy is Accuracy(correct: 28007, count: 50000, accuracy: 0.56014)
2019-02-19 05:50:53 INFO  DistriOptimizer$:152 - [Epoch 22 1026432/1281167][Iteration 19839][Wall Clock 57696.987160079s] Top1Accuracy is Accuracy(correct: 27948, count: 50000, accuracy: 0.55896)
2019-02-19 06:20:46 INFO  DistriOptimizer$:152 - [Epoch 23 618112/1281167][Iteration 20459][Wall Clock 59530.926688496s] Top1Accuracy is Accuracy(correct: 28280, count: 50000, accuracy: 0.5656)
2019-02-19 06:50:40 INFO  DistriOptimizer$:152 - [Epoch 24 209792/1281167][Iteration 21079][Wall Clock 61324.656562819s] Top1Accuracy is Accuracy(correct: 28296, count: 50000, accuracy: 0.56592)
2019-02-19 07:20:33 INFO  DistriOptimizer$:152 - [Epoch 24 1082752/1281167][Iteration 21699][Wall Clock 63077.743777083s] Top1Accuracy is Accuracy(correct: 28303, count: 50000, accuracy: 0.56606)
2019-02-19 07:50:28 INFO  DistriOptimizer$:152 - [Epoch 25 674432/1281167][Iteration 22319][Wall Clock 64912.009888737s] Top1Accuracy is Accuracy(correct: 28415, count: 50000, accuracy: 0.5683)
2019-02-19 08:20:24 INFO  DistriOptimizer$:152 - [Epoch 26 266112/1281167][Iteration 22939][Wall Clock 66708.507463808s] Top1Accuracy is Accuracy(correct: 29153, count: 50000, accuracy: 0.58306)
2019-02-19 08:50:21 INFO  DistriOptimizer$:152 - [Epoch 26 1139072/1281167][Iteration 23559][Wall Clock 68465.635101277s] Top1Accuracy is Accuracy(correct: 28516, count: 50000, accuracy: 0.57032)
2019-02-19 09:20:10 INFO  DistriOptimizer$:152 - [Epoch 27 730752/1281167][Iteration 24179][Wall Clock 70293.956236067s] Top1Accuracy is Accuracy(correct: 28939, count: 50000, accuracy: 0.57878)
2019-02-19 09:50:02 INFO  DistriOptimizer$:152 - [Epoch 28 322432/1281167][Iteration 24799][Wall Clock 72086.63694816s] Top1Accuracy is Accuracy(correct: 28991, count: 50000, accuracy: 0.57982)
2019-02-19 10:19:57 INFO  DistriOptimizer$:152 - [Epoch 28 1195392/1281167][Iteration 25419][Wall Clock 73841.794868842s] Top1Accuracy is Accuracy(correct: 28906, count: 50000, accuracy: 0.57812)
2019-02-19 10:49:56 INFO  DistriOptimizer$:152 - [Epoch 29 787072/1281167][Iteration 26039][Wall Clock 75680.315422568s] Top1Accuracy is Accuracy(correct: 29148, count: 50000, accuracy: 0.58296)
2019-02-19 11:19:50 INFO  DistriOptimizer$:152 - [Epoch 30 378752/1281167][Iteration 26659][Wall Clock 77474.408295804s] Top1Accuracy is Accuracy(correct: 29308, count: 50000, accuracy: 0.58616)
2019-02-19 11:49:41 INFO  DistriOptimizer$:152 - [Epoch 30 1251712/1281167][Iteration 27279][Wall Clock 79226.170741428s] Top1Accuracy is Accuracy(correct: 29452, count: 50000, accuracy: 0.58904)
2019-02-19 12:19:27 INFO  DistriOptimizer$:152 - [Epoch 31 843392/1281167][Iteration 27899][Wall Clock 81051.655202293s] Top1Accuracy is Accuracy(correct: 29508, count: 50000, accuracy: 0.59016)
2019-02-19 12:49:20 INFO  DistriOptimizer$:152 - [Epoch 32 435072/1281167][Iteration 28519][Wall Clock 82844.243723081s] Top1Accuracy is Accuracy(correct: 29771, count: 50000, accuracy: 0.59542)
2019-02-19 13:19:05 INFO  DistriOptimizer$:152 - [Epoch 33 26752/1281167][Iteration 29139][Wall Clock 84629.709764204s] Top1Accuracy is Accuracy(correct: 29716, count: 50000, accuracy: 0.59432)
2019-02-19 13:48:55 INFO  DistriOptimizer$:152 - [Epoch 33 899712/1281167][Iteration 29759][Wall Clock 86379.846532226s] Top1Accuracy is Accuracy(correct: 29865, count: 50000, accuracy: 0.5973)
2019-02-19 14:18:41 INFO  DistriOptimizer$:152 - [Epoch 34 491392/1281167][Iteration 30379][Wall Clock 88206.031214358s] Top1Accuracy is Accuracy(correct: 30065, count: 50000, accuracy: 0.6013)
2019-02-19 14:48:31 INFO  DistriOptimizer$:152 - [Epoch 35 83072/1281167][Iteration 30999][Wall Clock 89996.060527139s] Top1Accuracy is Accuracy(correct: 30200, count: 50000, accuracy: 0.604)
2019-02-19 15:18:29 INFO  DistriOptimizer$:152 - [Epoch 35 956032/1281167][Iteration 31619][Wall Clock 91754.175201837s] Top1Accuracy is Accuracy(correct: 29795, count: 50000, accuracy: 0.5959)
2019-02-19 15:48:28 INFO  DistriOptimizer$:152 - [Epoch 36 547712/1281167][Iteration 32239][Wall Clock 93591.655020198s] Top1Accuracy is Accuracy(correct: 30148, count: 50000, accuracy: 0.60296)
2019-02-19 16:18:21 INFO  DistriOptimizer$:152 - [Epoch 37 139392/1281167][Iteration 32859][Wall Clock 95385.420144594s] Top1Accuracy is Accuracy(correct: 30039, count: 50000, accuracy: 0.60078)
2019-02-19 16:48:18 INFO  DistriOptimizer$:152 - [Epoch 37 1012352/1281167][Iteration 33479][Wall Clock 97141.863405155s] Top1Accuracy is Accuracy(correct: 30122, count: 50000, accuracy: 0.60244)
2019-02-19 17:18:14 INFO  DistriOptimizer$:152 - [Epoch 38 604032/1281167][Iteration 34099][Wall Clock 98977.461572166s] Top1Accuracy is Accuracy(correct: 30392, count: 50000, accuracy: 0.60784)
2019-02-19 17:48:10 INFO  DistriOptimizer$:152 - [Epoch 39 195712/1281167][Iteration 34719][Wall Clock 100774.245717462s] Top1Accuracy is Accuracy(correct: 30325, count: 50000, accuracy: 0.6065)
2019-02-19 18:18:13 INFO  DistriOptimizer$:152 - [Epoch 39 1068672/1281167][Iteration 35339][Wall Clock 102537.478215084s] Top1Accuracy is Accuracy(correct: 29955, count: 50000, accuracy: 0.5991)
2019-02-19 18:48:13 INFO  DistriOptimizer$:152 - [Epoch 40 660352/1281167][Iteration 35959][Wall Clock 104376.742872373s] Top1Accuracy is Accuracy(correct: 30862, count: 50000, accuracy: 0.61724)
2019-02-19 19:18:00 INFO  DistriOptimizer$:152 - [Epoch 41 252032/1281167][Iteration 36579][Wall Clock 106164.398957351s] Top1Accuracy is Accuracy(correct: 30654, count: 50000, accuracy: 0.61308)
2019-02-19 19:47:53 INFO  DistriOptimizer$:152 - [Epoch 41 1124992/1281167][Iteration 37199][Wall Clock 107918.320872044s] Top1Accuracy is Accuracy(correct: 30774, count: 50000, accuracy: 0.61548)
2019-02-19 20:17:45 INFO  DistriOptimizer$:152 - [Epoch 42 716672/1281167][Iteration 37819][Wall Clock 109749.138154224s] Top1Accuracy is Accuracy(correct: 30580, count: 50000, accuracy: 0.6116)
2019-02-19 20:47:33 INFO  DistriOptimizer$:152 - [Epoch 43 308352/1281167][Iteration 38439][Wall Clock 111537.459995654s] Top1Accuracy is Accuracy(correct: 30609, count: 50000, accuracy: 0.61218)
2019-02-19 21:17:16 INFO  DistriOptimizer$:152 - [Epoch 43 1181312/1281167][Iteration 39059][Wall Clock 113281.437531819s] Top1Accuracy is Accuracy(correct: 30819, count: 50000, accuracy: 0.61638)
2019-02-19 21:47:10 INFO  DistriOptimizer$:152 - [Epoch 44 772992/1281167][Iteration 39679][Wall Clock 115114.574149746s] Top1Accuracy is Accuracy(correct: 30818, count: 50000, accuracy: 0.61636)
2019-02-19 22:16:54 INFO  DistriOptimizer$:152 - [Epoch 45 364672/1281167][Iteration 40299][Wall Clock 116897.798621315s] Top1Accuracy is Accuracy(correct: 30937, count: 50000, accuracy: 0.61874)
2019-02-19 22:46:50 INFO  DistriOptimizer$:152 - [Epoch 45 1237632/1281167][Iteration 40919][Wall Clock 118655.227215834s] Top1Accuracy is Accuracy(correct: 31021, count: 50000, accuracy: 0.62042)
2019-02-19 23:16:39 INFO  DistriOptimizer$:152 - [Epoch 46 829312/1281167][Iteration 41539][Wall Clock 120483.276990072s] Top1Accuracy is Accuracy(correct: 30892, count: 50000, accuracy: 0.61784)
2019-02-19 23:46:35 INFO  DistriOptimizer$:152 - [Epoch 47 420992/1281167][Iteration 42159][Wall Clock 122279.26256141s] Top1Accuracy is Accuracy(correct: 30853, count: 50000, accuracy: 0.61706)
2019-02-20 00:16:18 INFO  DistriOptimizer$:152 - [Epoch 48 12672/1281167][Iteration 42779][Wall Clock 124063.080924315s] Top1Accuracy is Accuracy(correct: 31178, count: 50000, accuracy: 0.62356)
2019-02-20 00:46:12 INFO  DistriOptimizer$:152 - [Epoch 48 885632/1281167][Iteration 43399][Wall Clock 125817.377722159s] Top1Accuracy is Accuracy(correct: 31177, count: 50000, accuracy: 0.62354)
2019-02-20 01:15:59 INFO  DistriOptimizer$:152 - [Epoch 49 477312/1281167][Iteration 44019][Wall Clock 127643.196631426s] Top1Accuracy is Accuracy(correct: 31106, count: 50000, accuracy: 0.62212)
2019-02-20 01:45:44 INFO  DistriOptimizer$:152 - [Epoch 50 68992/1281167][Iteration 44639][Wall Clock 129428.668127658s] Top1Accuracy is Accuracy(correct: 31109, count: 50000, accuracy: 0.62218)
2019-02-20 02:15:34 INFO  DistriOptimizer$:152 - [Epoch 50 941952/1281167][Iteration 45259][Wall Clock 131178.901671715s] Top1Accuracy is Accuracy(correct: 31432, count: 50000, accuracy: 0.62864)
2019-02-20 02:45:23 INFO  DistriOptimizer$:152 - [Epoch 51 533632/1281167][Iteration 45879][Wall Clock 133007.898611116s] Top1Accuracy is Accuracy(correct: 31367, count: 50000, accuracy: 0.62734)
2019-02-20 03:15:13 INFO  DistriOptimizer$:152 - [Epoch 52 125312/1281167][Iteration 46499][Wall Clock 134797.947921822s] Top1Accuracy is Accuracy(correct: 31413, count: 50000, accuracy: 0.62826)
2019-02-20 03:45:09 INFO  DistriOptimizer$:152 - [Epoch 52 998272/1281167][Iteration 47119][Wall Clock 136554.729137189s] Top1Accuracy is Accuracy(correct: 31336, count: 50000, accuracy: 0.62672)
2019-02-20 04:14:54 INFO  DistriOptimizer$:152 - [Epoch 53 589952/1281167][Iteration 47739][Wall Clock 138378.872425799s] Top1Accuracy is Accuracy(correct: 31785, count: 50000, accuracy: 0.6357)
2019-02-20 04:44:45 INFO  DistriOptimizer$:152 - [Epoch 54 181632/1281167][Iteration 48359][Wall Clock 140169.942951128s] Top1Accuracy is Accuracy(correct: 31861, count: 50000, accuracy: 0.63722)
2019-02-20 05:14:34 INFO  DistriOptimizer$:152 - [Epoch 54 1054592/1281167][Iteration 48979][Wall Clock 141919.365627635s] Top1Accuracy is Accuracy(correct: 31815, count: 50000, accuracy: 0.6363)
2019-02-20 05:44:22 INFO  DistriOptimizer$:152 - [Epoch 55 646272/1281167][Iteration 49599][Wall Clock 143746.88199014s] Top1Accuracy is Accuracy(correct: 31976, count: 50000, accuracy: 0.63952)
2019-02-20 06:14:06 INFO  DistriOptimizer$:152 - [Epoch 56 237952/1281167][Iteration 50219][Wall Clock 145530.619210967s] Top1Accuracy is Accuracy(correct: 31856, count: 50000, accuracy: 0.63712)
2019-02-20 06:43:53 INFO  DistriOptimizer$:152 - [Epoch 56 1110912/1281167][Iteration 50839][Wall Clock 147277.483938697s] Top1Accuracy is Accuracy(correct: 32084, count: 50000, accuracy: 0.64168)
2019-02-20 07:13:41 INFO  DistriOptimizer$:152 - [Epoch 57 702592/1281167][Iteration 51459][Wall Clock 149105.137682372s] Top1Accuracy is Accuracy(correct: 32085, count: 50000, accuracy: 0.6417)
2019-02-20 07:43:35 INFO  DistriOptimizer$:152 - [Epoch 58 294272/1281167][Iteration 52079][Wall Clock 150899.027090021s] Top1Accuracy is Accuracy(correct: 32191, count: 50000, accuracy: 0.64382)
2019-02-20 08:13:30 INFO  DistriOptimizer$:152 - [Epoch 58 1167232/1281167][Iteration 52699][Wall Clock 152654.423378576s] Top1Accuracy is Accuracy(correct: 32236, count: 50000, accuracy: 0.64472)
2019-02-20 08:43:25 INFO  DistriOptimizer$:152 - [Epoch 59 758912/1281167][Iteration 53319][Wall Clock 154488.72815275s] Top1Accuracy is Accuracy(correct: 32230, count: 50000, accuracy: 0.6446)
2019-02-20 09:13:21 INFO  DistriOptimizer$:152 - [Epoch 60 350592/1281167][Iteration 53939][Wall Clock 156285.224997442s] Top1Accuracy is Accuracy(correct: 32254, count: 50000, accuracy: 0.64508)
2019-02-20 09:43:11 INFO  DistriOptimizer$:152 - [Epoch 60 1223552/1281167][Iteration 54559][Wall Clock 158035.907639455s] Top1Accuracy is Accuracy(correct: 32322, count: 50000, accuracy: 0.64644)
2019-02-20 10:13:05 INFO  DistriOptimizer$:152 - [Epoch 61 815232/1281167][Iteration 55179][Wall Clock 159868.840341024s] Top1Accuracy is Accuracy(correct: 32457, count: 50000, accuracy: 0.64914)
2019-02-20 10:42:56 INFO  DistriOptimizer$:152 - [Epoch 62 406912/1281167][Iteration 55799][Wall Clock 161660.532341753s] Top1Accuracy is Accuracy(correct: 32619, count: 50000, accuracy: 0.65238)
2019-02-20 11:12:44 INFO  DistriOptimizer$:152 - [Epoch 62 1279872/1281167][Iteration 56419][Wall Clock 163408.27524626s] Top1Accuracy is Accuracy(correct: 32784, count: 50000, accuracy: 0.65568)
2019-02-20 11:42:44 INFO  DistriOptimizer$:152 - [Epoch 63 871552/1281167][Iteration 57039][Wall Clock 165248.056338252s] Top1Accuracy is Accuracy(correct: 32723, count: 50000, accuracy: 0.65446)
2019-02-20 12:12:34 INFO  DistriOptimizer$:152 - [Epoch 64 463232/1281167][Iteration 57659][Wall Clock 167038.079218206s] Top1Accuracy is Accuracy(correct: 32906, count: 50000, accuracy: 0.65812)
2019-02-20 12:42:27 INFO  DistriOptimizer$:152 - [Epoch 65 54912/1281167][Iteration 58279][Wall Clock 168831.375238254s] Top1Accuracy is Accuracy(correct: 33030, count: 50000, accuracy: 0.6606)
2019-02-20 13:12:22 INFO  DistriOptimizer$:152 - [Epoch 65 927872/1281167][Iteration 58899][Wall Clock 170585.847369645s] Top1Accuracy is Accuracy(correct: 33021, count: 50000, accuracy: 0.66042)
2019-02-20 13:42:19 INFO  DistriOptimizer$:152 - [Epoch 66 519552/1281167][Iteration 59519][Wall Clock 172423.479868469s] Top1Accuracy is Accuracy(correct: 33198, count: 50000, accuracy: 0.66396)
2019-02-20 14:12:16 INFO  DistriOptimizer$:152 - [Epoch 67 111232/1281167][Iteration 60139][Wall Clock 174220.36312542s] Top1Accuracy is Accuracy(correct: 33334, count: 50000, accuracy: 0.66668)
2019-02-20 14:42:16 INFO  DistriOptimizer$:152 - [Epoch 67 984192/1281167][Iteration 60759][Wall Clock 175980.816985268s] Top1Accuracy is Accuracy(correct: 33564, count: 50000, accuracy: 0.67128)
2019-02-20 15:12:13 INFO  DistriOptimizer$:152 - [Epoch 68 575872/1281167][Iteration 61379][Wall Clock 177816.729353829s] Top1Accuracy is Accuracy(correct: 33773, count: 50000, accuracy: 0.67546)
2019-02-20 15:42:08 INFO  DistriOptimizer$:152 - [Epoch 69 167552/1281167][Iteration 61999][Wall Clock 179612.085513965s] Top1Accuracy is Accuracy(correct: 34148, count: 50000, accuracy: 0.68296)
```

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/inception/ImageNet2012.scala [hunk 1]
```diff
@@ -81,3 +83,29 @@ object ImageNet2012 {
       ))
   }
 }
+
+
+object ImageNet2012Val {
```
Comment (zhichao-li) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/inception/ImageNet2012.scala]: How about we just pass a flag to `ImageNet2012` for `CropCenter`(i.e `train/test`) as it is the only difference between `ImageNet2012Val` and `ImageNet2012` ?

**Comment by zhichao-li** [comment]
Maybe we should remove the `Hflip` as well, take a look at: https://github.com/tensorflow/models/blob/master/research/inception/inception/image_processing.py
```
def eval_image(image, height, width, scope=None):
  """Prepare one image for evaluation.
  Args:
    image: 3-D float Tensor
    height: integer
    width: integer
    scope: Optional scope for name_scope.
  Returns:
    3-D float Tensor of prepared image.
  """
  with tf.name_scope(values=[image, height, width], name=scope,
                     default_name='eval_image'):
    # Crop the central region of the image with an area containing 87.5% of
    # the original image.
    image = tf.image.central_crop(image, central_fraction=0.875)

    # Resize the image to the original height and width.
    image = tf.expand_dims(image, 0)
    image = tf.image.resize_bilinear(image, [height, width],
                                     align_corners=False)
    image = tf.squeeze(image, [0])
return image
```

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/inception/ImageNet2012.scala (hunk 1 above)
Comment (qiuxin2012) [reply zoo/src/main/scala/com/intel/analytics/zoo/examples/inception/ImageNet2012.scala]: I think we can keep this.
We have different rdd name, crop method  and hflip, it won't be easy to read as we need to add a if else to the method.

**Comment by qiuxin2012** [comment]
jenkins passed

**Pull request merged**